### PR TITLE
Github-42969:fixes typo in table 5

### PR DESCRIPTION
**Pull request by JoeAldinger**
4.8+ 

Github issue #42969

[Preview](https://deploy-preview-43698--osdocs.netlify.app/openshift-enterprise/latest/networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.html)

@weliang1 can you take a look at this please for QE ack?

I'll need labels and merge. Please and thanks.

### CODE REVIEW

File: modules/nw-ovn-kubernetes-migration-about.adoc [hunk 1]
```diff
@@ -189,7 +189,7 @@ CNO:: Performs the following actions:
 |
 Reboot each node in the cluster.
 |
-Cluster:: As nodes reboot, the cluster assigns IP addresses to pods on the OVN-Kubernetes cluster network.
+Cluster:: As nodes reboot, the cluster assigns IP addresses to pods on the Openshift-SDN network.
```
Comment (tmalove) [inline modules/nw-ovn-kubernetes-migration-about.adoc:192]: @JoeAldinger I still see the former version of the sentence in the preview '...OVN-Kubernetes cluster...' 
https://deploy-preview-43698--osdocs.netlify.app/openshift-enterprise/latest/networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.html

Let me know if you see the updated content. Thanks!

Comment (JoeAldinger) [reply modules/nw-ovn-kubernetes-migration-about.adoc:192]: Hi Tami, 

I see "OVN-Kubernetes..." in table 4, but in table 5 I see "Openshift-SDN...". Is that what you see, if not let me know thanks.

Comment (ahardin-rh) [reply modules/nw-ovn-kubernetes-migration-about.adoc:192]: @JoeAldinger @tmalove The preview seems out of date, but I am fetching Joe's branch and building it locally to test and confirm. I will add an updated preview link manually before I merge.

**Pull request merged**